lib/vector-core/src/event/lua/mod.rs

```diff
@@ -1,6 +1,6 @@
+pub mod event;
 /// This module contains implementations of `ToLua` and `FromLua` traits for
 /// Vector data types
-pub mod event;
```

Comment:
This seems like an accidental change?

Reply:
Hmm, either accidental or rustfmt gore, yes.